Makefile.include

```diff
@@ -904,7 +904,7 @@ include $(RIOTMAKE)/tests/tests.inc.mk
 .PHONY: fuzz
 fuzz:
 	env FLASHFILE="$(FLASHFILE)" PORT="$(PORT)" TERMFLAGS="$(TERMFLAGS)" \
-		"$(RIOTBASE)"/dist/tools/fuzzing/afl.sh $(AFL_FLAGS)
+		"$(RIOTBASE)"/dist/tools/fuzzing/afl.sh $(FLAGS_FOR_AFL)
```

Comment:
Could you elaborate why renaming the environment variable is necessary?

Reply:
Sure! 

If you try to compile using AFL++ (instead of the old AFL) you will encounter this warning:
```
[!] WARNING: Mistyped AFL environment variable: AFL_FLAGS=
"make" -C RIOT/core/lib
Did you mean AFL_AS?
Did you mean AFL_CC?
```
This was introduced in Version ++3.10c of AFL++:
`printing suggestions for mistyped AFL_ env variables`
Check the changelog of AFL++ [here](https://github.com/AFLplusplus/AFLplusplus/blob/stable/docs/Changelog.md).

I am aware that RIOTs fuzzing documentation states to use the old AFL 2.52b - where this warning isn't present. However, AFL is no longer maintained. Tho, we should move on towards AFL++. So far all my fuzzing with AFL++ is without issues and the backwards compatibility is nice. This warning being the only issue.

Reply:
Moving to AFL++ is definitely a good idea! Maybe it also makes sense to update the documentation in this regard. However, I also wouldn't mind doing that in a separate merge request.